ui/app/models/replication-attributes.js

```diff
@@ -28,6 +28,14 @@ export default Fragment.extend({
       ? 'bootstrapping'
       : (this.get('isSecondary') && 'secondary') || (this.get('isPrimary') && 'primary');
   }),
+  modeForHeader: computed('mode', function() {
+    const mode = this.get('mode');
```

Comment:
nit: i finally looked this up since we use `this.get('stuff')` and `this.stuff` throughout the codebase and i couldn't remember why, but as of Ember 3.1x we can use `this.stuff`: https://blog.emberjs.com/2018/02/16/ember-3-1-beta-released.html

Reply:
Thank you!  I've wondered the same.